doc/classes/Environment.xml

```diff
@@ -9,7 +9,11 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
-		These effects will only apply when the [Viewport]'s intended usage is "3D" or "3D Without Effects". This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property.
+		In "2D Without Sampling", all post-processing effects will be turned off. In "3D Without Effects", these options will be invalid:
```

Comment:
```suggestion
		With "2D Without Sampling", all post-processing effects will be turned off. With "3D Without Effects", these options will be invalid:
```

Reply:
> I think this would be clearer, everything else looks good to me, thanks :)

thank u 2

Reply:
What @mhilbrunner meant is that you should update this PR to improve the text as suggested. You can do so with `git commit --amend` and then force push to update the PR while keeping a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).